src/components/z-divider/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, Prop, h, Host } from "@stencil/core";
+import { DividerSize, DividerOrientation } from "../../beans";
+@Component({
+  tag: "z-divider",
+  styleUrl: "styles.css",
+  shadow: false,
+  scoped: true,
+})
+export class ZDivider {
+  /** [optional] Divider size */
+  @Prop() size?: DividerSize = DividerSize.small;
+  /** [optional] Divider color */
+  @Prop() color?: string = "gray200";
+  /** [optional] Divider orintation */
+  @Prop() orientation?: DividerOrientation = DividerOrientation.horizontal;
+
+  render() {
+    return (
+      <Host
+        class={`divider-${this.size} divider-${this.orientation}`}
+        style={{ backgroundColor: `var(--${this.color})` }}
```

Comment:
I would suggest to set the color using a css var instead of a prop. It can be set in the `:host` general style and then used in the component stylesheet. Any product can easily override it.
Something like this:
```css
:host {
  --divider-color: var(--gray200);
  
  ...
}
```
Then the product can do:
```css
  z-divider {
    --divider-color: var(--red500);
  }
```
Works well with `shadow: true` but `scoped: true` needs a more specific selector, like `z-divider[class*='divider-']` because of selectors priority.

Some component is already using this method.

Reply:
Al momento abbiamo concordato di mantenere il color token come prop. Al prossimo incontro dev si discuterà sulla best practice da utilizzare in sviluppi futuri